authors/tests/test_articles.py

```diff
@@ -31,3 +31,12 @@ def test_update_article(self):
         self.create_article()
         response = self.update_article()
         self.assertEqual(response.status_code, 201)
+
+class BookMarkTest(BaseTest):
+
+
+    def test_bookmark_article(self):
+        response = self.bookmark_article()
+        print(response)
```

Comment:
I know that `print`  is a savior in bad times, but  you forgot to remove it when things started working

Reply:
Noted